### PR TITLE
About page

### PR DESCRIPTION
**Pull request by gustavoguichard**
sobre nos tem acucar

sobre nos tem acucar 2

sobre nos tem acucar 3

### CODE REVIEW

**Comment by gustavoguichard** [comment]
This PR has the code for `/share` page as well. #6

File: app/views/pages/sharelove.html.slim [hunk 1]
```diff
@@ -102,7 +102,7 @@
         p.lead.dark-gray
           strong #ficaadica :
           |  Comprando os adesivos você colabora com a gente pra que o Tem Açúcar fique cada vez melhor e crie vizinhanças mais felizes e sustentáveis 😉
-        / Botão pra comprar o adesivo( disponível só no Brasil)
+        a.button.large.primary href="https://benfeitoria.com/temacucar" target="_blank" Comprar adesivos
```
Comment (gustavoguichard) [inline app/views/pages/sharelove.html.slim]: screen shot 2016-10-14 at 12 53 35

**Comment by gustavoguichard** [comment]
Link to "download stickers"
screen shot 2016-10-14 at 12 53 35

**Comment by danielweinmann** [comment]
@gustavoguichard can you let me know when you are finished so I can review?

**Comment by gustavoguichard** [comment]
👌

**Pull request merged**